lib/tools/adb-emu-commands.js

```diff
@@ -2,7 +2,14 @@ import log from '../logger.js';
 import _ from 'lodash';
 
 const PHONE_NUMBER_PATTERN = /^[\+]?[(]?[0-9]*[)]?[-\s\.]?[0-9]*[-\s\.]?[0-9]{2,}$/im;
+
 let emuMethods = {};
+emuMethods.POWER_AC_ON = 'on';
```

Comment:
In Appium we have no usually kept constants on objects. Instead we just export the constants from the module and allow folks to import them when they need them.
```js
const POWER_AC_ON = 'on';

let emuMethods = {};

// ...

export default emuMethods;
export { emuMethods, POWER_AC_ON };
```